frontend/scripts/react-components/profiles/place-summary.component.jsx

```diff
@@ -49,9 +46,6 @@ function PlaceSummary(props) {
     { name: capitalize(biomeName), label: 'Biome' },
     { name: capitalize(stateName), label: 'State' }
   ];
-  const commodityAreaValue = formatValue(commodityArea, 'area');
-  const areaValue = formatValue(area, 'area');
-  const commodityProductionValue = formatValue(commodityProduction, 'tons');
```

Comment:
I think that if commodityProduction is tons here, the new value in the constants should also be tons.